AndelaOSP/wire
# #155720290-Move mock_endpoints folder

#### What does this PR do?
Places the mock endpoints file in the correct directory
#### Description of Task to be completed?
The mock endpoints folder has been moved out of src and placed in the root directory of the project 
#### How should this be manually tested?
The app can be run using `yarn start` after all dependencies have been installed and see that the mock data is showing on dashboard and timelines 
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#155720290
#
